pyproject.toml

```diff
@@ -681,7 +682,6 @@ ignore = [
     "COM812",
     "COM819",
     "ISC001",
```

Comment:
What about this rule? Or is there a reason to keep it ignored?

Reply:
It's a rule that collides with the formatter, that's why it's disabled